Cargo.toml

```diff
@@ -53,6 +53,7 @@ rusoto_s3 = "0.47.0"
 serde = "1.0.125"
 serde_json = "1.0.64"
 serde_urlencoded = "0.7.0"
+urlencoding = "2.1.0"
```

Comment:
Use `cargo sort` to keep lexicographical order.

<https://crates.io/crates/cargo-sort>